tsup.config.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from 'tsup';
+
+// eslint-disable-next-line import/no-default-export
+export default defineConfig(({ watch }) => ({
+  entry: ['src/index.ts'],
+  format: ['esm', 'cjs'],
+  outDir: 'build',
+  target: 'node16',
```

Comment:
Could we use node18? Given that 16 support is ending next year

Reply:
I don't think that it's a good idea right now. Because most likely not everyone who is using SDK already migrated to node 18 including us. :)